updates/7/updates.sql

```diff
@@ -50,5 +50,9 @@ UPDATE assets SET coingecko="ndex" where identifier="_ceth_0x1966d718A565566e8E2
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x1966d718A565566e8E202792658D7b5Ff4ECe469", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x1966d718A565566e8E202792658D7b5Ff4ECe469", "C", "nDEX", "NDX", 1534287453, NULL, "ndex", "", "0x1966d718A565566e8E202792658D7b5Ff4ECe469");
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x383518188C0C6d7730D91b2c03a03C837814a899", 9, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x383518188C0C6d7730D91b2c03a03C837814a899", "C", "Olympus", "OHM", 1616366920, NULL, "olympus", "OHM", "0x383518188C0C6d7730D91b2c03a03C837814a899");
 *
-INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", 9, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", "C", "DAppNode DAO Token", "NODE", 1626520282, NULL, "dappnode", "", "0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0");
-*
\ No newline at end of file
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0", "C", "DAppNode DAO Token", "NODE", 1626520282, NULL, "dappnode", "", "0xDa007777D86AC6d989cC9f79A73261b3fC5e0DA0");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5B7533812759B45C2B44C19e320ba2cD2681b542", 8, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x5B7533812759B45C2B44C19e320ba2cD2681b542", "C", "SingularityNET Token", "AGIX", 1619584798, NULL, "singularitynet", "AGIX", "0x5B7533812759B45C2B44C19e320ba2cD2681b542");
+*
+UPDATE assets SET swapped_for="_ceth_0x5B7533812759B45C2B44C19e320ba2cD2681b542" WHERE identifier="_ceth_0x8eB24319393716668D768dCEC29356ae9CfFe285";
```

Comment:
Couldn';t youy also set cryptocompare for the old token here since it seems there is none? Aren't they one to one? Here and in the insertion below.